bpf-gpl/tc.c

```diff
@@ -680,7 +686,10 @@ int calico_tc_skb_accepted_entrypoint(struct __sk_buff *skb)
 		goto deny;
 	}
 	ip_header = skb_iphdr(skb);
-
+	if (ip_header->ihl > 5) {
```

Comment:
Is this one needed?  This is in the epilogue program so the packet should have already been dropped above if it had options.

Reply:
yes. This is not needed